tests/utils/signer.py

```diff
@@ -0,0 +1,179 @@
+# source: https://github.com/OpenZeppelin/cairo-contracts/tree/main/tests/signers.py
```

Comment:
`openzeppelin-cairo-contracts` is in the project requirements so we should not need to copy/paste

Reply:
Yes, but the scripts I am using are in their tests, so not in the package